arelle/plugin/validate/ESEF/ESEF_Current/ValidateXbrlFinally.py

```diff
@@ -855,7 +855,7 @@ def checkFootnote(elt: ModelInlineFootnote | ModelResource, text: str) -> None:
         unreportedLbElts = set()
         for arcroles, error, checkRoots, lbType in (
                     ((parentChild,), "elements{}UsedForTagging{}AppliedInPresentationLinkbase", True, "presentation"),
-                    ((summationItem,), "elements{}UsedForTagging{}AppliedInCalculationLinkbase", False, "calculation"),
+                    ((summationItems,), "elements{}UsedForTagging{}AppliedInCalculationLinkbase", False, "calculation"),
```

Comment:
Out of curiosity, why `(summationItems,)` rather than plain `summationItems`, and then below `if arcrole in (parentChild,) + summationItems`?  That would be consistent with `hc_all, hc_notAll,...` entry just below.

Reply:
There's no compelling reason. modelXbrl.relationshipSet can accept an arcrole or a tuple of arcroles. Both should behave the same.